### PR TITLE
Add holmes.js@1.16.6 w/ npm auto-update

### LINKED ISSUE #8244: [Request] Add Holmes
**Library name:** Holmes
**Git repository url:** https://github.com/Haroenv/holmes/
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #8244, cc @Haroenv

Pull request for issue: #8244

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Haroenv/holmes/
 * Official website (optional, not the repository): https://haroen.me/holmes/
 * NPM package url (optional): https://www.npmjs.com/package/holmes.js
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 23
   - Count of stars: 1,468
   - Count of forks: 75
 * NPM download stats (optional):
   - Downloads in the last day: 51
   - Downloads in the last week: 119
   - Downloads in the last month: 690

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/holmes/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "search",
+    "dom",
+    "filter"
+  ],
+  "npmName": "holmes.js",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "+(holmes|main)?(.es).js"
```
Comment (Haroenv) [inline ajax/libs/holmes/package.json]: Main isn't needed on a cdn

File: ajax/libs/holmes/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "filename": "holmes.js",
+  "keywords": [
+    "search",
+    "dom",
+    "filter"
+  ],
+  "npmName": "holmes.js",
+  "npmFileMap": [
+    {
+      "basePath": "js",
```
Comment (pvnr0082t) [inline ajax/libs/holmes/package.json]: It's not good to set `js` or `css` as basePath. Let's leave it blank. Thanks.

File: ajax/libs/holmes/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "files": [
+        "holmes?(.es).js"
```
Comment (pvnr0082t) [inline ajax/libs/holmes/package.json]: We can simply use `"holmes*"`.

File: ajax/libs/holmes/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "holmes",
```
Comment (pvnr0082t) [inline ajax/libs/holmes/package.json]: We can follow the name on NPM. Let's use `holmes.js`.

**Comment by pvnr0082t** [comment]
@PeterDaveHello The `holmes.js` before v1.13.5 are not minified, how could we handle it?

**Comment by Haroenv** [comment]
Why should deprecated versions be available in the first place? 😄

**Comment by pvnr0082t** [comment]
@Haroenv We will host all the versions of the library, so the old versions should be concerned.

**Comment by Haroenv** [comment]
I can’t publish an existing tag on npm again, so IDK how to handle this 😕

**Comment by PeterDaveHello** [comment]
@Haroenv I think old versions are not deprecated version, a little bit different

**Comment by PeterDaveHello** [comment]
We just try our best to host all the versions, but won't force to do it if there is a problem.

**Comment by PeterDaveHello** [comment]
For the not-minified old versions, I'll take care of them, that's fine.

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍

**Comment by pvnr0082t** [comment]
@dakshshah96 Test failed. Please try to fix it. :+1:

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍 I've modified the commit message too.

**Comment by PeterDaveHello** [comment]
@Haroenv I'm sure they are not, please use checksum to confirm that, they are not the files in the npm 1.16.6 release.

**Comment by Haroenv** [comment]
The difference is a newline.

**Comment by PeterDaveHello** [comment]
@Haroenv which will break the hash/integry, we should keep them totally the same as we can :)

**Comment by Haroenv** [comment]
Yes, I just wrote this because I found out what the source of the difference is.

**Comment by PeterDaveHello** [comment]
@Haroenv :+1:

**Comment by dakshshah96** [comment]
@PeterDaveHello Fixed the files. 👍

**Pull request merged**